jetstream/core/orchestrator.py

```diff
@@ -526,6 +548,15 @@ def _prefill_thread(self, idx: int):
       if self._metrics_collector:
         self._metrics_collector.get_request_input_length().observe(true_length)
 
+      if self._metrics_collector:
+        self._metrics_collector.get_time_per_prefill_token().observe(
+            (
+                request.metadata.transfer_enqueue_time
+                - request.metadata.prefill_dequeue_time
+            )
+            / true_length
+        )
+
```

Comment:
Could we wrap all the `if self._metrics_collector:` into functions in prometheus.py `JetstreamMetricsCollector` class? We could do it in the follow up PR.